scripts/ci/print.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+# be stricter
+set -eou pipefail
+
+# Print the public API diff from the cli tool in the Github workflow summary
+#
+# Takes the following function args:
+# $1 - outdated librustzcash dependency where the version is not latest, in format - "crate_name;..."
+# $2 - the URL, pointing to the Github workflow console output of the public API diff cli tool
+print_workflow_diff() {
+	local outdated_libs=$1
+	local diff_result_workflow_url=$2
+
+	echo -e "# :warning: New versions of librustzcash libraries are present :warning: \n"
+	echo -e "You can view a better colored result of the diff in the **[CI logs]($diff_result_workflow_url)**.\n"
+
+	IFS=';' read -ra arr <<<"$outdated_libs"
+	for lib_name in "${arr[@]}"; do
+		if [[ -z "$lib_name" ]]; then
+			continue
+		fi
+
+		local lib_latest_version
+		lib_latest_version=$(curl --silent "https://crates.io/api/v1/crates/$lib_name" | jq -r '.crate.max_stable_version')
+
+		local lib_current_version
+		lib_current_version=$(cargo metadata --format-version=1 -q --manifest-path=./uniffi-zcash-lib/lib/Cargo.toml | jq -r --arg lib_name "$lib_name" '.packages[] | select(.name == $lib_name) | .version')
+
+		echo "## ${lib_name}"
+		echo "\`CURRENTLY USED VERSION\`    :arrow_right: ${lib_current_version}"
+		echo "\`LATEST PUBLISHED VERSION\`  :arrow_right: ${lib_latest_version}"
+		echo ""
+		echo "\`\`\`diff"
+		cat "$lib_name".diff
+		echo ""
+		echo "\`\`\`"
+	done
+}
+
+# Print the public API diff from the cli tool in the Github workflow summary
+#
+# Takes the following function args:
+# $1 - a boolean string "true" or "false", indicating if the build failed
+# $2 - the URL, pointing to the Github workflow console output of the uniffi-zcash-lib build
+print_workflow_build_result() {
+	local build_failing=$1
+	local build_result_workflow_url=$2
+
+	if [[ "$build_failing" == "true" ]]; then
+		echo "# :warning: Build fails after bumping to the newer versions with the following output: :warning: "
+		echo "You can also view the build result in the **[CI logs]($build_result_workflow_url)**."
+		echo "\`\`\`"
+		output=$(cat build_output)
+		echo "$output"
```

Comment:
missing local but you could just do
```suggestion
		cat build_output
```